ignition-server/cmd/start.go

```diff
@@ -169,11 +185,16 @@ func run(ctx context.Context, opts Options) error {
 	go mgr.Start(ctx)
 
 	mgr.GetLogger().Info("Using opts", "opts", fmt.Sprintf("%+v", opts))
+	eventRecorder := mgr.GetEventRecorderFor("ignition-server")
 
 	mux := http.NewServeMux()
 	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
 		log.Printf("User Agent: %s. Requested: %s", r.Header.Get("User-Agent"), r.URL.Path)
 
+		tokenSecret := nodepool.TokenSecret(os.Getenv(namespaceEnvVariableName),
+			util.ParseNamespacedName(r.Header.Get("NodePool")).Name,
```

Comment:
What happens to instances that were created right before upgrading the hypershift operator that don't have these headers?